src/SAFE.Client/SAFE.fs

```diff
@@ -130,15 +157,21 @@ module RemoteData =
     /// Unwraps the Loaded value, or returns the default value.
     let defaultValue defaultValue (remote: RemoteData<'T>) = remote.DefaultValue defaultValue
 
-    /// Returns whether the `RemoteData<'T>` value has been loaded or not.
+    /// Returns whether the `RemoteData<'T>` value has been loaded.
     let hasLoaded (remote: RemoteData<'T>) = remote.HasLoaded
 
-    /// Returns whether the `RemoteData<'T>` value has started loading or not.
-    let hasStarted (remote: RemoteData<'T>) = remote.HasStarted
```

Comment:
Likewise, removing this is a breaking change, which could negatively impact consumers.